cscs-checks/microbenchmarks/gpu/gpu_burn/gpu_burn_test.py

```diff
@@ -133,3 +135,27 @@ def set_gpus_per_node(self):
             self.num_gpus_per_node = 3
         else:
             self.num_gpus_per_node = 1
+
+    @rfm.run_before('performance')
+    def report_smallest_node(self):
+        regex = r'\[(\S+)\] GPU\s+\d\(OK\): (\d+) GF/s'
+        rptf = os.path.join(self.stagedir, sn.evaluate(self.stdout))
+        self.nids = sn.extractall(regex, rptf, 1)
+        self.flops = sn.extractall(regex, rptf, 2, float)
+        index = -1
+        flops_min = sn.min(self.flops)
+        for ii in range(len(sn.evaluate(self.flops))):
+            if self.flops[ii] == flops_min:
+                index = ii
+                break
```

Comment:
```suggestion
        flops_min, index = sys.float_info.max, None
        for i, flops in enumerate(self.flops):
            if flops < flops_min:
                flops_min, index = flops, i
```

Reply:
:+1: 
I can't manage to call `self.flops.index(min(self.flops))`.
What's the right way to call a deferred function ?

Reply:
`index` is no method of `self.flops`, is it?

Reply:
If you want to call `index()`, you would have to do it like this: `self.flops.evaluate().index(min(self.flops))`.